thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/AnomalyDetectionInputContext.java

```diff
@@ -13,19 +13,19 @@
 
 
 public class AnomalyDetectionInputContext {
-  Map<DimensionMap, MetricTimeSeries> dimensionKeyMetricTimeSeriesMap = Collections.emptyMap();
+  Map<DimensionMap, MetricTimeSeries> dimensionMapMetricTimeSeriesMap = Collections.emptyMap();
```

Comment:
Is it absolutely necessary to rename the DimensionKeyMetric...Map to DimensionMapMetric...Map? if not let's keep it.